app/assets/images/payment_icons/axs.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="pi-Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" xml:space="preserve" viewBox="0 0 38 24" role="img" width="38" height="24" aria-labelledby="pi-axs"><title id="pi-axs">AXS</title><path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/><path d="M19.233 13.141c-.729.637-.741.649-1.391 1.29l.642.645h-.214c-1.391 0-4.289.037-4.289.037l-.908-.9-6.791.003-.963.86H3.714l.107-.108a1153.49 1153.49 0 005.905-6.014l2.664-.012s3.098 3.122 4.489 4.52l.066.079c.546-.518.931-.88 1.347-1.319-1.036-1.096-2.079-2.129-3.091-3.233.107 0 4.353-.043 4.353-.043s2.382 2.174 2.489 2.282l2.251-2.297s1.36.016 1.895.016l-.214.215c.535-.323.913-.258 1.226-.265 1.577-.034 4.242.036 6.169.036h.214v1.075c-1.391 0-2.686.014-4.077.014l.107.108c1.07.645 3.947 2.273 4.337 2.888.535.645.428 1.29-.321 1.72-.535.323-1.127.337-1.662.337l-10.799.001c-.673-.64-1.266-1.324-1.936-1.935zm6.11.991h.214c.535 0 3.027.024 3.883.024.107 0 .323 0 .321-.108-.002-.108-.091-.126-.214-.215-.194-.141-2.718-1.476-3.902-2.554-.177-.161-.403-.261-.516-.764-.047-.21-.035-.456.072-.564-.749.753-1.259 1.361-2.115 2.114.856.86 1.401 1.529 2.257 2.389v-.322zM7.46 13.141h4.602c-.84-.847-1.4-1.423-2.355-2.366-.75.809-1.374 1.454-2.247 2.366z" fill="#070809"/></svg>
```

Comment:
Can we remove these attributes `id="pi-Layer_1" x="0" y="0" xml:space="preserve"`? I don't think we need it.

Reply:
`<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24"...`

removed per your request